src/ostorlab/cli/scanner/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Module for the root command: scanner."""
```

Comment:
Can you please remove this code. Init import have caused many headaches in the past.

Reply:
It is the case for the all sub-commands.